config/settings/test.yml

```diff
@@ -225,6 +225,12 @@ lighthouse:
         client_id: abc123456
         rsa_key: path/to/key
       use_mocks: false
+  benefits_education: 
+      host: https://sandbox-api.va.gov
+      use_mocks: <% if ENV['DISABLE_MOCKS'] %>false<% else %>true<% end %>
+      access_token:
+        client_id: <% if ENV['DISABLE_MOCKS'] %><%= ENV['LIGHTHOUSE_BENEFITS_EDUCATION_CLIENT_ID'] || abc123456 %><% else %> abc123456 <% end %>
+        rsa_key: <% if ENV['DISABLE_MOCKS'] %><%= ENV['LIGHTHOUSE_BENEFITS_EDUCATION_RSA_KEY_PATH'] || path/to/key %><% else %> path/to/key <% end %>
```

Comment:
I updated this code, and will include this in my documentation request to update [VCR Debugging](https://depo-platform-documentation.scrollhelp.site/developer-docs/vcr-debugging) documentation on the docs site.  

It'd work like this:
```
export LIGHTHOUSE_BENEFITS_EDUCATION_CLIENT_ID=def7890
export LIGHTHOUSE_BENEFITS_EDUCATION_RSA_KEY_PATH=/Users/user1/.ssh/private.pem
```

Then to (re)generate the VCR cassette:
```
rm spec/support/vcr_cassettes/lighthouse/benefits_education/200_response.yml
DISABLE_MOCKS=1 rspec spec/lib/lighthouse/benefits_education/service_spec.rb
```

Reply:
@AdamKing0126 Can you replace the ruby syntax with a string some kind? The ENV variables can be added in the manifest repo later on

Reply:
```
        client_id: abc123456
        rsa_key: ~
```

Reply:
Hey @RachalCassity can you explain, "The ENV variables can be added in the manifest repo later on"

I made these changes so that when I was running the tests locally, i could supply credentials in order to make the network request.  If this is not the correct strategy, could you point me to the accepted one?

Reply:
I was curious about this portion as well, but it does seem to allow the tests to run successfully. 

Our team will typically add the actual values to the `test.yml` to record a cassette and just modify the `test.yml` and the cassette values back to the "mocked" or "acceptable" values after. I was more curious of the above to cut out those additional steps.

Reply:
you can also add filters to prevent any slip ups @AdamKing0126 - assuming you may be aware of this but FYSA: https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb

Reply:
@FonzMP Yeah that's the way I was originally doing it, but I was worried that I would accidentally commit my client_id.  I can definitely add an entry for these values so that doesn't happen.

I'm happy to update `test.yml` to just have `~` values instead of the Ruby-injected strings (though I have to admit I liked that pattern)

Reply:
@AdamKing0126 Will the tests work using `~` values?

Reply:
@RachalCassity yes because the tests don't normally in the course of things call out to an external service - they replay VCR cassettes.  But if you want to create a cassette or regenerate one, it means you've got to go in and modify test.yml to include your credentials (and then remember to remove them again).  Just seems a little unergonomic.

I added entries to the file that Alex linked me to, so at least I won't be nervous about committing my creentials to version control!